src/test/java/com/google/api/codegen/testdata/java_mock_service_impl_library.baseline

```diff
@@ -43,37 +43,38 @@ import com.google.protobuf.Empty;
 import com.google.protobuf.GeneratedMessage;
 import io.grpc.stub.StreamObserver;
 import java.util.ArrayList;
+import java.util.LinkedList;
 import java.util.List;
+import java.util.Queue;
 
 @javax.annotation.Generated("by GAPIC")
 public class MockLibraryServiceImpl implements LibraryService  {
   private ArrayList<GeneratedMessage> requests;
-  private ArrayList<GeneratedMessage> responses;
+  private Queue<GeneratedMessage> responses;
 
   public MockLibraryServiceImpl() {
     requests = new ArrayList<>();
-    responses = new ArrayList<>();
+    responses = new LinkedList<>();
   }
 
   public List<GeneratedMessage> getRequests() {
     return requests;
   }
 
   public void setResponses(List<GeneratedMessage> responses) {
```

Comment:
It makes sense to use varargs here. But in order to make the full change I need to update the interface in gax-java so I would prefer to address this in a separate PR. Issue filed in gax: https://github.com/googleapis/gax-java/issues/100